framework/DataObjects/XDataSet.py

```diff
@@ -526,6 +524,41 @@ def remove(self,realization=None,variable=None):
     #either way reset kdtree
     self.inputKDTree = None
 
+  def renameVariable(self,old,new):
+    """
+      Changes the name of a variable from "old" to "new".
+      @ In, old, str, old name
+      @ In, new, str, new name
+      @ Out, None
+    """
+    # determine where the old variable was
+    isInput = old in self._inputs
+    isOutput = old in self._outputs
+    isMeta = old in self._metavars
+    isIndex = old in self.indexes
+    # make the changes to the variable listings
+    if isInput:
+      self._inputs = list(a if (a != old) else new for a in self._inputs)
+    if isOutput:
```

Comment:
even if in terms of performance should be the same, I would prefer to have ```elif``` statements for more clarity of the flow logic.

Reply:
ok. no problem to me